core/deployment/src/main/java/io/quarkus/deployment/pkg/builditem/ModuleDirBuildItem.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkus.deployment.pkg.builditem;
+
+import java.nio.file.Path;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+public final class ModuleDirBuildItem extends SimpleBuildItem {
```

Comment:
Please add a class level JavaDoc, something like `A BuildItem holding the base directory of the current module (<code>${basedir}</code> for Maven projects or <code>${projectDir}</code> for Gradle projects).`